usr/share/rear/verify/CDM/default/410_use_replica_cdm_cluster_cert.sh

```diff
@@ -39,5 +39,8 @@ StopIfError "Could not copy replica CDM cluster certificate"
 
 /usr/bin/chmod $v 600 ${CDM_KEYS_DIR}/rubrik.crt
 
+mv $v ${CDM_KEYS_DIR}/agent.crt ${CDM_KEYS_DIR}/agent.crt.orig
+mv $v ${CDM_KEYS_DIR}/agent.pem ${CDM_KEYS_DIR}/agent.pem.orig
+/etc/rubrik/rba-keygen.sh
```

Comment:
@DamaniN  Why is this required?

Reply:
I added a LogPrintError message so theer is at least some user info.
That is all I could do because I do not have a Rubrik (CDM) cluster environment.